configs/config.d/redmine.conf

```diff
@@ -1,6 +1,7 @@
 <source>
   @type tail
   format none
+  # /var/log/redmine/default/production.log was empty after running 5 minutes.
```

Comment:
Interesting.

Looking at a few `production.log` samples in their forum, there are some patterns. But they are mixed together.

http://www.redmine.org/boards/2/topics/51723
http://www.redmine.org/boards/2/topics/51978

```
Started GET "/projects/messagerie-test/issues" for 192.168.10.20 at 2017-01-03 10:34:06 +0100
Processing by IssuesController#index as HTML
Parameters: {"project_id"=>"messagerie-bluemind"}
Current user: georges.robert (id=129)
Completed 500 Internal Server Error in 70ms (ActiveRecord: 28.6ms)

NoMethodError (undefined method `each' for nil:NilClass):
app/models/issue_query.rb:357:in `issues'
app/controllers/issues_controller.rb:70:in `index'
lib/redmine/sudo_mode.rb:63:in `sudo_mode'
plugins/redmine_dmsf/lib/redmine_dmsf/webdav/no_parse.rb:33:in `call'
```

```
Started GET "/redmine/users/531" for 172.18.140.18 at 2017-02-06 13:37:42 +0300
Processing by UsersController#show as HTML
Parameters: {"id"=>"531"}
Current user: seckin.aktas (id=531)
Rendered users/show.html.erb within layouts/base (101.6ms)
Completed 200 OK in 209ms (Views: 115.8ms | ActiveRecord: 29.5ms)
Started GET "/redmine/users/531/edit" for 172.18.140.18 at 2017-02-06 13:37:55 +0300
Processing by UsersController#edit as HTML
Parameters: {"id"=>"531"}
Current user: seckin.aktas (id=531)
Rendered plugins/redmine_digest/app/views/account_settings/_settings_2x.html.erb (1.8ms)
Rendered plugins/redmine_landing_page/app/views/users/_landing_page.html.erb (0.4ms)
Rendered users/_mail_notifications.html.erb (48.8ms)
Rendered users/_preferences.html.erb (25.1ms)
Rendered users/_form.html.erb (86.3ms)
Rendered users/_general.html.erb (87.5ms)
Rendered users/_groups.html.erb (7.9ms)
Rendered principal_memberships/_index.html.erb (298.7ms)
Rendered users/_memberships.html.erb (299.1ms)
Rendered common/_tabs.html.erb (396.5ms)
Rendered users/edit.html.erb within layouts/admin (399.9ms)
Rendered admin/_menu.html.erb (4.2ms)
Rendered layouts/base.html.erb (29.3ms)
Completed 200 OK in 443ms (Views: 431.9ms | ActiveRecord: 6.1ms)

Started PATCH "/redmine/users/531" for 172.18.140.18 at 2017-02-06 13:38:05 +0300
Processing by UsersController#update as HTML
Parameters: {"utf8"=>"✓", "authenticity_token"=>"KaL/dY5t2nARw3achYyxod7M6nlijTPovob0jv7YcspmWiBio9kQW29IpZGLxWt46USpHafXpweRyfOll4E6Xg==", "user"=>{"login"=>"seckin.aktas", "firstname"=>"Seckin", "lastname"=>"Aktas", "mail"=>"xxxx", "language"=>"tr", "digest_account_attributes"=>{"active"=>"1", "id"=>"61"}, "landing_page"=>"", "auth_source_id"=>"1", "password"=>"[FILTERED]", "password_confirmation"=>"[FILTERED]", "generate_password"=>"[FILTERED]", "must_change_passwd"=>"0", "mail_notification"=>"only_my_events", "notified_project_ids"=>[""]}, "pref"=>{"no_self_notified"=>"0", "hide_mail"=>"1", "time_zone"=>"Istanbul", "comments_sorting"=>"asc", "warn_on_leaving_unsaved"=>"1"}, "commit"=>"Kaydet", "id"=>"531"}
Current user: seckin.aktas (id=531)
Completed 500 Internal Server Error in 11ms (ActiveRecord: 3.0ms)

ActiveModel::ForbiddenAttributesError (ActiveModel::ForbiddenAttributesError):
lib/redmine/safe_attributes.rb:84:in `safe_attributes='
app/controllers/users_controller.rb:135:in `update'
lib/redmine/sudo_mode.rb:63:in `sudo_mode'
```

I'm fine if we leave it to the users to decide exactly how to handle this.